blessings/__init__.py

```diff
@@ -0,0 +1,560 @@
+"""A thin, practical wrapper around terminal coloring, styling, and
```

Comment:
Sorry, I added this in to make the terminal output nicer without asking the user to install python dependencies.

I can scrap it if it's annoying, but I thought it was nice. I always like colourful prompts that look like someone put thought and effort into them myself.